model/work.py

```diff
@@ -737,7 +737,18 @@ def all_editions(self, policy=None):
         )
         return q
 
-    def all_identifier_ids(self, policy=None):
+    def all_identifier_ids(self, policy=None, unified=True):
+        """Return all Identifier IDs associated with this Work.
+
+        :param policy: A `PresentationCalculationPolicy`.
+        :return: If unified=False, a 2-tuple (`direct_ids`, `all_ids`).
+           `direct_ids` is a list of Identifier IDs associated with
+             the Work's LicensePools.
```

Comment:
The process for deriving `direct_ids` is _just_ complicated enough that I decided to change the return value of this method so I could reuse it rather than copy-and-pasting the code (if it was less complicated) or refactoring it (if it was more complicated). I'm happy to do either instead.

For clarity, here's the code:

```
  primary_identifier_ids = [
            lp.identifier.id for lp in self.license_pools
            if lp.identifier
=]
```

Reply:
I'd have copied it just to make this method easier to understand, but it's fine either way. It's a little weird that one of the return values is a list and one is a set though.

Reply:
OK, I'll copy it to simplify the API.